### PR TITLE
Move request to dev dependencies

### PR DESCRIPTION
**Pull request by TrySound**
Request package was left in dependencies here https://github.com/nylas/nylas-nodejs/pull/234

# License
I confirm that this contribution is made under the terms of the MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by TrySound** [comment]
cc @mrashed-dev

File: package.json [hunk 1]
```diff
@@ -68,6 +67,7 @@
     "jest": "^24.9.0",
     "prettier": "^1.16.4",
     "pretty-format": "^25.2.6",
+    "request": "^2.88.0",
```
Comment (mrashed-dev) [inline package.json:70]: Do we need to actually keep this? I'm fine with taking it out completely, not even having it as a dev dependency.  Same with line 60, `"@types/request": "^2.48.4"`, we can probably take both lines out!

**Review by mrashed-dev — COMMENTED** [review]
Good catch! Left a comment let me know what you think @TrySound

File: package.json (hunk 1 above)
Comment (TrySound) [reply package.json:70]: I still see it in tests

Comment (TrySound) [reply package.json:70]: Ah, I see. Unused in tests :)

Comment (TrySound) [reply package.json]: Removed

**Review by mrashed-dev — APPROVED** [review]
Great catch, thanks for the PR @TrySound!

**Pull request merged**